content/integrations/cloud_foundry.md

```diff
@@ -0,0 +1,283 @@
+      traffic_controller_url: <LOGGREGATOR_URL> # e.g. ws://traffic-controller.your-cf-domain.com:8081
+    nozzle:
+      deployment: <DEPLOYMENT_NAME>    # tags each firehose metric with 'deployment:<DEPLOYMENT_NAME>'
+      subscription_id: datadog-nozzle  # can be anything (firehose streams data evenly to all jobs using the same subscription_id)
+      # disable_access_control: true   # for development only
+      # insecure_ssl_skip_verify: true # for development only; enable if your UAA does not use a verifiable cert
+    uaa:
+      client: datadog-firehose-nozzle # client name you just configured
+      client_secret: <SECRET_YOU_JUST_CONFIGURED>
+      url: <UAA_URL> # e.g. https://uaa.your-cf-domain.com:8443
+~~~
+
+You can set `metrics_prefix` to anything you like, but if you want to use the default Cloud Foundry screenboard and to see metrics metadata in dashboards and graphs, set it to `cloudfoundry.nozzle.`.
+
+See [all configuration options](https://github.com/DataDog/datadog-firehose-nozzle-release/blob/master/jobs/datadog-firehose-nozzle/spec) for the Datadog Firehose Nozzle.
+
+In the same manifest, add the Datadog Nozzle release name and version:
+
+~~~
+releases:
+# - name: some-other-release
+#   version: x.y.z
+# ...
+  - name: datadog-firehose-nozzle
+    version: $VERSION_YOU_UPLOADED # i.e. 61
+~~~
+
+### Redeploy the deployment
+
+~~~
+# BOSH CLI v1
+bosh deployment cf-manifest.yml
+bosh -n deploy
+
+# BOSH CLI v2
+bosh -n -d cf-manifest deploy cf-manifest.yml
+~~~
+
+### Verify the Nozzle is collecting
+
+On the [Metrics explorer](https://app.datadoghq.com/metric/explorer) page in Datadog, search for metrics beginning with the prefix you configured (i.e. `cloudfoundry.nozzle`):
+
+![cloud-foundry-nozzle-metrics](/static/images/cloud-foundry-nozzle-metrics.png)
+
+# Configure the Datadog plugin for BOSH Health Monitor
+
+This plugin is packaged with the BOSH Health Monitor, so if the Health Monitor is already installed and running on your BOSH Director, you simply need to configure the plugin and redeploy the Director.
+
+### Add the configuration
+
+In the manifest originally used to deploy the Director (e.g. bosh.yml), configure the Datadog plugin:
+
+~~~
+instance_groups:
+- instances: 1
+  properties:
+    hm:
+    # loglevel: info
+    # resurrector: enabled
+    #...other stuff
+      datadog_enabled: true
+      datadog:
+        api_key: <YOUR_DATADOG_API_KEY>
+        application_key: <YOUR_DATADOG_APP_KEY>
+        pagerduty_service_name: "your_service_name" # optional
+~~~
+
+### Redeploy the Director
+
+BOSH cannot simply configure the plugin and restart the Health Monitor; it must redeploy the Director entirely. **To do this without without fully wiping the Director—including its persistent disks and its PostgreSQL database—you MUST have a `state.json` (or similarly named) file that accurately reflects the current state of your Director.**
```

Comment:
This is my understanding from the [BOSH docs](https://bosh.io/docs/cli-envs.html#deployment-state), but I haven't confirmed that for sure it will destroy ones disks.

If what is written there in bold is true, then it *should* scare users. Would we rather have weaker language that results in someone unwittingly blowing away their Director's database?